submissions/peterIyiola/DOM/signup.js

```diff
@@ -0,0 +1,70 @@
+let submitForm = document.getElementById('validateForm'),
+email = document.getElementById('email'),
+firstName = document.getElementById('firstName'),
+lastName = document.getElementById('lastName'),
+password = document.getElementById('password'),
+confirmPassword = document.getElementById('confirmPassword'),
+phoneNumber = document.getElementById('phoneNumber');
+
+const emailRegex = /^\w+([-]?\w+)*@\w+([-]?\w+)*(\.\w{2,10})+$/;
+const nameRegex = /^[a-zA-Z]/;
+const phoneNumberRegex = /^[+]?[0-9]{8,14}$/i;
+const passwordRegex = /(?=.*\d)(?=.*[a-z])(?=.*[a-zA-Z]).{6,}$/;
+
+
+//names,phone number, email input field function declaration
+function validation(field, regex, event, result, error1) {
+if (field.value === '') {
+    field.nextElementSibling.innerHTML = result;
+    event.preventDefault();
+    return false;
+} else if (!field.value.match(regex)) {
+    field.nextElementSibling.innerHTML = error1;
+    event.preventDefault();
+    return false;
+} else {
+    field.nextElementSibling.innerHTML = "";
+    return true;
+}
+}
+//password function declaration
+function validatePassword(password, confirmPassword, event) {
```

Comment:
The first 3 checks on this function is irrelevant, kindly still use the `validation()` for them